repos/cli-utils/src/network/getPublicIPsForUrl.js

```diff
@@ -0,0 +1,17 @@
+const dns = require('dns')
+
+/**
+ * @param {String} url 
+ * @returns {Promise} a promise that resolves to the ip address mapped to the url
+ */
+const getPublicIPsForUrl = url => {
```

Comment:
Is this actually being used anywhere?
* I'm not against it, I'm just curious for why it was added.